packages/frontend/src/pages/EmailVerification.tsx

```diff
@@ -0,0 +1,147 @@
+import { Intent } from '@blueprintjs/core';
+import React, { FC, useState } from 'react';
+import Countdown, { zeroPad } from 'react-countdown';
+import { Helmet } from 'react-helmet';
+import { useForm } from 'react-hook-form';
+import AnchorLink from '../components/common/AnchorLink';
+import Page from '../components/common/Page/Page';
+import {
+    PasswordInputField,
+    SubmitButton,
+} from '../components/CreateUserForm/CreateUserForm.styles';
+import PageSpinner from '../components/PageSpinner';
+import Form from '../components/ReactHookForm/Form';
+import { useApp } from '../providers/AppProvider';
+import LightdashLogo from '../svgs/lightdash-black.svg';
+import {
+    AnchorLinkWrapper,
+    CardWrapper,
+    ErrorMessage,
+    FooterCta,
+    FormFooterCopy,
+    FormWrapper,
+    Logo,
+    LogoWrapper,
+    Subtitle,
+    Title,
+} from './SignUp.styles';
+
+const EmailVerification: FC<{
+    nbrOfAttempts: number;
+    expirationTime: string;
+    // future time '2020-02-01T01:02:03'
+    onSubmit: () => void;
+    onResend: () => void;
+}> = ({ nbrOfAttempts, expirationTime, onSubmit, onResend }) => {
+    const { health } = useApp();
```

Comment:
I guess this is just for testing? It would be good to separate out the component out from the page logic. So removing helmet etc. and having two components: the email verification form component, and the page that contains the component